src/api/java/com/minecolonies/api/colony/permissions/IPermissions.java

```diff
@@ -37,6 +37,14 @@
     @Nullable
     String getOwnerName();
 
+    /**
+     * Checks if a user is a subscriber.
+     *
+     * @param player {@link EntityPlayer} to check for subscription.
+     * @return True is subscriber, otherwise false.
+     */
+    boolean isSubscriber(@NotNull EntityPlayer player);
```

Comment:
Does this now also check the `Global Subscribers`?

Reply:
no it is unrelated to the actual subscribers and just describes a boolean of the ranks which say whether the rank can receive views. Only pulled this one up to the interface, but might be a good idea to rename